ui/components/app/dropdowns/dropdown.js

```diff
@@ -18,7 +18,7 @@ export class Dropdown extends Component {
       borderRadius: '4px',
       padding: '8px 16px',
       background: 'var(--color-background-default)',
-      boxShadow: 'rgba(0, 0, 0, 0.15) 0px 2px 2px 2px',
+      boxShadow: 'var(--shadow-size-sm) var(--color-shadow-default)',
       ...innerStyle,
     };
 
```

Comment:
This component is used in the network dropdown

## Before
<img width="414" alt="Screen Shot 2022-07-20 at 9 15 52 AM" src="https://user-images.githubusercontent.com/8112138/180032251-ea5c891b-da0b-4410-9516-fc69e8ca996c.png">
<img width="408" alt="Screen Shot 2022-07-20 at 9 16 11 AM" src="https://user-images.githubusercontent.com/8112138/180032278-e52e3536-d742-42cb-b047-76e310084088.png">

## After
<img width="406" alt="Screen Shot 2022-07-20 at 9 16 45 AM" src="https://user-images.githubusercontent.com/8112138/180032398-3e327392-2ec6-4e55-8f5b-c42cc83c5c43.png">
<img width="378" alt="Screen Shot 2022-07-20 at 9 16 26 AM" src="https://user-images.githubusercontent.com/8112138/180032381-6370f62c-d65f-45f6-8e46-6c8d93ea34ea.png">